apps/communications/contacts/views/details/js/boot.js

```diff
@@ -0,0 +1,58 @@
+/* global DetailsUI, Details, LazyLoader, fbLoader, ContactsService */
+'use strict';
+
+window.addEventListener('DOMContentLoaded', function() {
+  LazyLoader.load([
+    '/shared/js/l10n.js'], function() {
+    LazyLoader.load([
+      document.getElementById('view-contact-details')
+    ], function() {
+      // TODO Add if needed
+    });
+  });
+});
+
+window.onload = function() {
+  var dependencies = [
+    '/contacts/style/webrtc-client/webrtc_client.css',
+    '/contacts/js/webrtc-client/webrtc_client.js',
```

Comment:
Do we need this at the start? I guess we can make the buttons appear if we detect them after the first paint.